Header header

string id
float64 current
float64 out_voltage
float64 batt_voltage
	
